subcommand/server-acl-init/command_test.go

```diff
@@ -129,7 +116,7 @@ func TestRun_Defaults(t *testing.T) {
 func TestRun_TokensPrimaryDC(t *testing.T) {
```

Comment:
the doc block needs to be updated. Remove the "We test using the -release-name flag vs using the
// -resource-prefix flag."